### ISSUE TITLE
can anyone guide to do quantization for custom trained yolov8

### ISSUE THREAD

**Issue by 218w1d7706** [ultralytics/ultralytics#12768]
please suggest some sources. i have tried several sources but nothing works for me.

**Comment by glenn-jocher** [comment]
Hello! For quantizing a custom-trained YOLOv8 model, you can use the export functionality with INT8 quantization. Here's a simple example on how to export your model to TensorRT format with INT8 precision:

```bash
yolo export model=path/to/your/custom_model.pt format=engine int8=True
```

Make sure to perform this on the same device you plan to deploy the model, as INT8 calibration is device-specific. For more detailed guidance, you can refer to the [TensorRT integration documentation](https://docs.ultralytics.com/integrations/tensorrt/) provided by Ultralytics.

If you encounter any specific issues during this process, feel free to share them here for more targeted assistance! 🚀

**Comment by the0807** [comment]
I'm trying to reduce the loss, but try it
<https://github.com/the0807/YOLOv8-ONNX-TensorRT>

**Comment by glenn-jocher** [comment]
Hello! To reduce the loss during training, ensure your dataset is well-prepped and consider tweaking hyperparameters like learning rate or batch size. Also, using a pre-trained model can provide a good starting point. For specific adjustments in loss, reviewing the training logs to understand where the model might be underperforming can be helpful. If you're looking into using TensorRT for optimization, ensure your model is properly calibrated, especially when using INT8 precision. Good luck! 🚀
